app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+
+def copy_file(command: str) -> None:
+    new_str = command.split()
+    cmd, file_first, file_second = new_str
```

Comment:
what if the command is "cp file1.txt file2.txt file3.txt"? unpacking will fail

Reply:
yes, you are right, but I can't unpacking after if. So, I need return to my first way with index or divide if to two parts?